hare/hare.go

```diff
@@ -278,6 +295,9 @@ var (
 // GetResult returns the hare output for the provided range.
 // Returns error iff the request for the upper is too old.
 func (h *Hare) GetResult(lid types.LayerID) ([]types.BlockID, error) {
+
+	h.With().Warning("GetResult", lid)
```

Comment:
I assume this is a warning for debugging purposes - let's change to info/debug (feels like a debug to me, but up to you).